apps/src/ReadENVI.cpp

```diff
@@ -3,19 +3,64 @@
 //
 
 #include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
 #include <opencv2/core.hpp>
 #include <opencv2/imgcodecs.hpp>
 
 #include "envitools/ENVI.hpp"
 
 namespace et = envitools;
 namespace fs = boost::filesystem;
+namespace po = boost::program_options;
 
-int main(int /*argc*/, char* argv[])
+int main(int argc, char** argv)
 {
+    fs::path hdrPath;
+
+    // clang-format off
+    po::options_description options("Options");
+    options.add_options()
+        ("help,h","Show this message")
+        ("hdr-path,p", po::value<std::string>()->required(),
+            "Path of ENVI file")
+        ("band,b", po::value<int>()->required(),
```

Comment:
I want to be able to specify a list of bands. `-b 0,27,180` would export the 0, 27, and 180th band. `-b all` would extract all of the bands. You probably will need a function to split by comma (I think I have an example of this), then you'll need to make sure the given bands are actually within the range of possibilities. Conversion will then need to be a loop for each band.